perfkitbenchmarker/pkb.py

```diff
@@ -138,7 +138,12 @@
     'Type for all scratch disks. The default is standard')
 flags.DEFINE_integer('scratch_disk_iops', 1500,
                      'IOPS for Provisioned IOPS (SSD) volumes in AWS.')
-
+flags.DEFINE_integer('num_striped_drives', 1,
```

Comment:
Maybe `num_scratch_disks`, if this determines the number of remote disks created?
It doesn't solve the local disk case, but this name doesn't seem intuitive to me.